src/main/java/marquez/db/Columns.java

```diff
@@ -0,0 +1,60 @@
+package marquez.db;
+
+import java.sql.Array;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.time.Instant;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.UUID;
+
+public final class Columns {
+  public static final String ROW_UUID = "guid";
```

Comment:
nit: might be good to alphabetize this list.

Reply:
Good point. I'll break down the list by table: `namespace`, `owner`, etc